.github/workflows/ci-arm64.yml

```diff
@@ -68,6 +68,7 @@ jobs:
               --add-module=./modules/ngx_http_concat_module \
               --add-module=./modules/ngx_http_footer_filter_module \
               --add-module=./modules/ngx_http_lua_module \
+              --add-module=./modules/ngx_http_lua_upstream \
```

Comment:
hi @lhanjian I have examined the test cases introduced by this module. I believe it is easy to run and pass, it does not depend on some other module or library. Could you try to make it work in our CI/CD workflow?

Reply:
@chobits OK. ok i will try it soon

Reply:
@chobits `ngx_http_lua_upstream` need more dependencies, like `echo-nginx-module`.

and version of `test-nginx`  is too old to run `ngx_http_lua_upstream/t`, so we can't run test of  `ngx_http_lua_upstream/t` for now.